<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Web Applications for Life Sciences</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/scrolling-nav.css" rel="stylesheet">

</head>

<body id="page-top">

<!-- Navigation -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
    <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">WALS@ICWE2023</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#topics">Topics of Interest</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#submission">Submission</a>
                </li>
                <!--<li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#special_issue">Special Issue</a>
                </li>-->
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#dates">Important Dates</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#committee">Committee</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#program">Program</a>
                </li>
                <!--<li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#collaborations">Collaborations</a>
                </li>-->
            </ul>
        </div>
    </div>
</nav>

<header class="text-black-50" style="background-image: url('imgs/scenario_bg.jpg'); min-height: 27rem;">
    <div class="container text-center">
        <h1>The 2<sup>nd</sup> International Workshop <br> on Web Applications for Life Sciences</h1>

        <img src="imgs/logo2023.png" alt="Logo WALS" class="logo-wals" width="320">

        <br><br>
        <h3>6th June, 2023 in Alicante, Spain</h3>
        <!--<h4 style="color:red;opacity: 0.7;">Hybrid workshop (in person/virtually)</h4>-->
        <!--<h3 style="color:red">Held in person</h3>-->

        <br>
        <h4>In conjunction with the 23<sup>rd</sup> International Conference on Web Engineering (ICWE&nbsp;2023) - 6-9th
            June</h4>
    </div>
</header>

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">We are giving a <b>10-days extension</b> for the last ones! New hard deadline is
                    <b>August 7th 2020</b>.</p>
                <p class="text-justify lead"> Important info:
                <ul class="lead">
                    <li>All accepted papers will be published by Springer in the LNCS series in the usual book that
                        collects all ER Workshop contributions
                    </li>
                    <li>We invite all papers accepted to the workshop to submit a revised and extended version to the
                        <a href="https://bmcbioinformatics.biomedcentral.com">BMC Bioinformatics journal</a> for a post-conference
                        supplement related to Conceptual Modeling in Life Sciences.
                    </li>
                <li>For the <b>best paper of the workshop</b> (as assessed by the evaluations of the members of the Program
                        Committee) the <b>Article Processing Charges of the BMC Bioinformatics supplement will be fully
                        covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at Universitat Politècnica de València)
                    </li>
                </ul>
                </p>
                <p class="text-justify lead">
                    The 1st International Workshop on Conceptual Modeling for Life Sciences is going virtual, together
                    with the main conference <a href="https://er2020.big.tuwien.ac.at/"> ER 2020</a>.
                    We will soon provide further details on the platforms we will use and the
                    necessary logistics.
                    An important note upfront – there will be free registration for non-authors, and a heavily reduced
                    registration fee for authors presenting a paper. Registration with further details will be open on
                    August 15, 2020.</p>
            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>
                <p class="text-justify lead"> We made a compilation of the workshop presentations. You can watch it <a href="https://www.youtube.com/watch?v=-l4OUdrkVC4&list=PLfWxoOMC6swI59BOYNw0U3L2VNJ_dqSBG&ab_channel=bioinformaticspolimi">here on YouTube</a>.
                <p class="text-justify lead">Check also the Program (at the end of the page) for slides and videos of each video (we will keep loading them as we get permission from the authors).</p>

            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead"><b>November 4th, 2021</b>: our BMC Bioinformatics Supplement on CMLS 2020 (Part 1) is finally out!
    Check it out here <a
                        href="https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13">
                    https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13</a>.
                Keep connected to know when also Part 2 is out. Also for CMLS 2021 we are planning a supplement on
                    the same Journal. The deadline for extended articles (invited) is Jan. 31st, 2022.</p>
                 <p class="text-justify lead"><b>October 19th, 2021</b>: Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>

            </div>

        </div>
    </div>
</section>-->


<section id="about">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>About</h2>
                <p class="text-justify lead">
                    The recent advances in unraveling the secrets of human conditions and diseases have encouraged new
                    paradigms for their diagnosis and treatment. In addition, the recent pandemics have brought
                    increasing attention toward the genetic mechanisms of viruses and infected hosts. The information
                    related to these phenomena is increasing at unprecedented rates, directly impacting the design and
                    development of data management pipelines and their applications. New ways of processing and exposing
                    data and knowledge on the Web in healthcare and life sciences environments are thus strongly needed.
                    <br><br>
                    The WALS workshop, at its second edition, aims at being a meeting forum for Web Engineering, Data
                    Management, and Bioinformatics researchers working on life sciences problems, offering the
                    opportunity to share, discuss and find new approaches to support Search, Knowledge Extraction, and
                    Data Science on the Web, thereby achieving important results for healthcare, precision medicine,
                    biology, genomics, and virology.
                    <br><br>
                    The workshop welcomes submissions of fresh investigations and demonstrations concerning conceptual,
                    experimental, and applied studies of life sciences problems on the Web. Directions of particular
                    interest include: 1) monitoring and adaptation of Web Engineering techniques for life sciences; 2)
                    design and engineering of Web applications for resolving life sciences problems; 3) development of
                    crowdsourcing and collaborative strategies for life sciences data curation, integration, and
                    discovery.


                </p>
            </div>
        </div>
    </div>
</section>

<section id="topics">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Topics of interest</h2>
                <p class="text-justify lead">
                    The workshop focuses on Web Engineering as a means for facing the challenges that emerge when
                    designing and developing applications for life sciences, focused on several branches such as
                    healthcare, precision medicine, molecular biology, genomics, and virology. The workshop is not
                    restricted to particular research methods, and we will consider conceptual, empirical, and applied
                    research (including practical demonstrations). The topics of interest include, but are not limited
                    to:


                </p>
                <p class="text-justify lead">The topics of interest include, but are not limited to:</p>
                <ul class="lead">
                    <li>Web applications modeling and engineering for genomics</li>
                    <li>Web applications for the management of electronic/digital health records (EHR)</li>
                    <li>Web user interfaces for genomics, virology, or molecular biology</li>
                    <li>Development of Web systems for Precision Medicine</li>
                    <li>Web services for exploration of life sciences databases and knowledge graphs</li>
                    <li>Biological knowledge extraction</li>
                    <li>Genomic big data analytics</li>
                    <li>Web crowdsourcing methods for life sciences data curation and annotation</li>
                    <li>Information visualization and exploratory analysis methods for biology</li>
                    <li>Data science applications to healthcare and biology on the Web</li>
                    <li>Semantic Web knowledge representation for genomics</li>
                    <li>Life sciences Semantic Web applications</li>
                    <li>Biomedical ontologies engineering</li>
                    <li>Interoperability of health information systems and databases on the Web</li>
                    <li>Web Engineering techniques for FAIRification of biological datasets</li>
                    <li>COVID-19 and SARS-CoV-2 tracking via Web applications</li>
                </ul>
            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for WALS 2022:</p>
                <ul class="lead">
                    <li>Marco Antonio Tangaro, Marica Antonacci, Pietro Mandreoli, Daniele Colombo, Nadina Foggetti,
                        Giacinto Donvito, Graziano Pesole and Federico Zambelli.
                        <i>The Laniakea Dashboard and storage encryption components: a foundation for developing
                            on-demand cloud services for Life Science information on submission</i></li>
                    <li>Alberto García, Ana León, Mireia Costa, Oscar Pastor, Daniel González-Ibea, Estela Pérez-Román,
                        Carles Borredà, Javier Terol, Victoria Ibanez, Francisco R. Tadeo and Manuel Talón.
                        <i>CitrusGenome: A Bioinformatics Tool to Characterize, Visualize, and Explore Large Citrus
                            Variant
                            Datasets information on submission</i></li>
                    <li>Paolo Perlasca, Marco Frasca, Cheick Tidiane Ba, Jessica Gliozzo, Marco Notaro, Mario
                        Pennacchioni,
                        Giorgio Valentini and Marco Mesiti.
                        <i>Integration and Visual Analysis of Biomolecular Networks</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<section id="submission">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Paper submission guidelines</h2>
                <p class="text-justify lead">
                    We invite submissions of high quality papers describing original and unpublished results
                    regarding any of the workshop’s topics of interest.
                    <br><br>
                    Accepted papers will be included in the post-workshop proceedings to appear in Springer's series,
                    as per the usual tradition of the ICWE conference joint events (see last year's <a
                        href="https://link.springer.com/book/10.1007/978-3-031-25380-5">ICWE 2022 Workshops</a>).

                    <br><br>
                    The authors must submit manuscripts using the Springer-Verlag LNCS style for Lecture Notes in
                    Computer Science.
                    For style files and details, see the page <a
                        href="https://www.springer.com/gp/computer-science/lncs/conference-proceedings-guidelines">https://www.springer.com/gp/computer-science/lncs/conference-proceedings-guidelines</a>.
                    The workshop will accept two types of contributions:
                <ul class="lead">
                    <li>Full papers - maximum 12 pages including references</li>
                    <li>Short/demo papers - maximum 6 pages including references</li>
                </ul>
                <p class="text-justify lead">
                    Papers must be submitted as PDF files using EasyChair at <a
                        href="https://easychair.org/conferences/?conf=wals2023" target="_blank">https://easychair.org/conferences/?conf=wals2023</a>.
                    <br><br>
                    To ensure high quality, all papers will be thoroughly peer reviewed by the Program Committee.
                    Manuscripts not submitted in the LNCS style or having more than 12 pages will be automatically
                    rejected.
                    The papers need to be original and not submitted or accepted for publication in any other
                    workshop, conference, or journal.
                    <br><br>
                    For each accepted paper, at least one author must attend the workshop and present the paper.
                    Registration is subject to the terms and conditions of ICWE.
                    <br><br>
                    Main contact: <b>algarsi3@pros.upv.es</b>
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="cameraready">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <strike><h2>Camera-ready guidelines</h2></strike>
                <strike><p class="text-justify lead">
                    For the preparation of their manuscript camera-ready version, authors should consult <a
                        href="ftp://ftp.springernature.com/cs-proceeding/svproc/guidelines/Springer_Guidelines_for_Authors_of_Proceedings.pdf">Springer’s
                    authors’ guidelines</a>
                    and use their <a
                        href="https://www.springer.com/de/it-informatik/lncs/conference-proceedings-guidelines">proceedings
                    templates</a>, either for LaTeX or for Word,
                    for the preparation of their papers. The page limit for workshop papers is 10 pages, as for the
                    previously submitted version.
                    Springer encourages authors to include their ORCIDs in their papers.
                    <br><br>
                    In addition, the corresponding author of each paper, acting on behalf of all of the authors of that
                    paper,
                    must complete and sign a <a
                        href="https://drive.google.com/file/d/1FL5_jFrqayqbPRZ2nTuNKaY55_8L8xD8/view?usp=sharing">Consent-to-Publish
                    form</a>.
                    The corresponding author signing the copyright form
                    should match the corresponding author marked on the paper.
                    Once the files have been sent to Springer, changes relating to the authorship of the papers cannot
                    be made.
                </p></strike>
            </div>
        </div>
    </div>
</section>
-->


<!--<section id="special_issue">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Post-conference publication</h2>
                <p class="text-justify lead">
                    The papers accepted to the workshop, will be published within a Springer CCIS series volume
                    as per the usual tradition of the ICWE conference joint events (see last year's <a
                        href="https://link.springer.com/book/10.1007/978-3-031-25380-5">ICWE 2022 Workshops</a>).
                    <br>
                    In addition, they will be invited to submit a revised and extended version
                    for a post-conference supplement in the journal
                    <a href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC Bioinformatics</b></a>
                    or a similar venue (e.g., <a href="https://bmcmedinformdecismak.biomedcentral.com/" target="_blank"><b>BMC
                    Medical Informatics and Decision Making</b></a>),
                    depending on the topic.
                </p>
                <p>
                    The authors of accepted papers interested in submitting an extended article to a BMC
                    supplement will need to fill, sign, scan and submit the supplement submission letter of
                    interest.

                    This document is important for us so we can know how many extended articles we will receive.
                    <br>
                    The authors can read the BMC Bioinformatics guidelines <a
                        href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines"
                        target="_blank">here</a> and the BMC Medical Informatics and
                    Decision Making guidelines <a
                        href="https://bmcmedinformdecismak.biomedcentral.com/submission-guidelines"
                        target="_blank">here</a>.
                    There are no page limits for the BMC journals.
                    The articles can be submitted on the <a hred="https://www.editorialmanager.com/supp/"
                                                            target="_blank">BMC Supplements website</a>.
                    <br>
                    The submitted extended articles will go through two review phases.
                    In the first phase, the organizers of the WALS 2022 workshop will serve as guest editors.
                    Some additional colleagues might be involved as additional guest editors, in case of high number of
                    submissions.
                    At the end of the first review phase, the guest editors will recommend the submitted extended
                    articles for acceptance or rejection,
                    and transfer them to the BMC journals' editor-in-chiefs who will take care of the second review
                    phase.
                    <br>
                    The BMC editor-in-chiefs can decide to confirm the guest editors' recommendations or to start a new
                    review phase (with new reviewers invited, new reviews, new requests, etc).
                    The BMC editor-in-chief will eventually make the final decision on the acceptance or rejection of
                    each extended manuscript.
                    <br>
                    Please note that, even if the guest editors suggest the acceptance for a specific paper, the BMC
                    editor-in-chief can still decide to reject it (this case is quite rare but it can still happen).
                    <br>

                </p>
            </div>
        </div>
    </div>
</section>-->


<section id="dates">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Important dates (AoE)</h2>
                <p class="lead">
                <ul class="lead">
                    <li>Paper submission deadline: <strike>April 14th, 2023</strike> <b>April 21st, 2023</b> (extended!)
                    </li>
                    <li>Notification deadline: May 5th, 2023</li>
                    <li>Article revision deadline: May 15th, 2023</li>
                    <li>Workshop day: June 6th, 2023</li>
                    <li>Camera-ready deadline: July 9th</li>
                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<section id="registration">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Registration</h2>
                <p class="lead">
                    Please refer to the main conference website <a
                        href="https://icwe2023.webengineering.org/registration/">here</a>.
                    At least one of the authors for each accepted paper must have a <b>Regular Registration"</b>
                    and attend the workshop day, for paper presentation.
                </p>
            </div>
        </div>
    </div>
</section>

<section id="organizers">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Organizers</h2>
                <p class="lead">
                <div class="row">
                    <div class="col-md-4 d-flex justify-content-center">
                        <a href="https://annabernasconi.faculty.polimi.it/"><img src="imgs/anna2022.JPG"
                                                                                 alt="annabernasconi"
                                                                                 width="190"></a><br>
                    </div>
                    <div class="col">
                        <b>Anna Bernasconi</b>
                        <br>
                        Department of Electronics, Information, and Bioengineering - Politecnico di Milano,
                        Italy
                        <br>anna.bernasconi@polimi.it

                        </p>
                        <p class="text-justify">
                            Anna Bernasconi is a Research Fellow in the Department of Electronics, Information, and
                            Bioengineering also at Politecnico di Milano. Her research focuses on conceptual modeling,
                            data integration, semantic web, and biological data analysis. Since the COVID-19 pandemic,
                            her research has moved to viral genomics, by building models, databases, and Web search
                            systems for viral sequences and their variants. She co-organized and chaired the first three
                            International Workshops on Conceptual Modeling for Life Sciences (CMLS 2020 / 2021 / 2022)
                            co-located with the ER conference and co-organized the First International Workshop on Web
                            Applications for Life Sciences (WALS 2022) co-located with the ICWE conference. <!-- She is
                            Program Chair of IEEE Big Data Service 2023 and Special Track Chair and Proceedings Chair of
                            IEEE Big Data Service 2022. She is Publicity Chair for ER 2023. Further information can be
                            found at https://annabernasconi.faculty.polimi.it/.-->
                        <p>
                    </div>
                </div>


                <p class="lead">
                <div class="row">
                    <div class="col-md-4 d-flex justify-content-center">
                        <img src="imgs/alberto2023.jpeg"
                             alt="albertogarcia"
                             width="180" height="195"><br>
                    </div>
                    <div class="col">
                        <b>Alberto García S.</b>
                        <br>Valencian Research Institute for Artificial Intelligence, Universitat Politecnica de
                        Valencia, Spain.
                        <br>algarsi3@pros.upv.es
                        </p>
                        <p class="text-justify">
                            Alberto García S. obtained his Ph.D. cum laude at Universitat Politecnica de Valencia under
                            the supervision of Prof. Oscar Pastor. He is currently a researcher at the VRAIN research
                            institute, where he focuses on conceptual modeling, data science, and User Interface design.
                            He has participated in multiple research projects in collaboration with clinicians and
                            geneticists to generate knowledge from genomics data effectively and efficiently.

                        </p>
                    </div>
                </div>
                <p class="lead">
                <div class="row">
                    <div class="col-md-4 d-flex justify-content-center">
                        <img src="imgs/pietro2022.png" alt="pietropinoli" width="200" height="200"><br>
                    </div>
                    <div class="col">
                        <b>Pietro Pinoli</b>
                        <br>Department of Electronics, Information, and Bioengineering - Politecnico di Milano,
                        Italy
                        <br>pietro.pinoli@polimi.it
                        </p>
                        <p class="text-justify">
                            Pietro Pinoli works as Research Fellow and lecturer at the Department of Electronics,
                            Information and Bioengineering at the Politecnico di Milano (Italy). He received his PhD cum
                            laude in 2017, with a thesis titled “Modeling and Querying Genomic Data” where he proposed
                            and benchmarked data structures and algorithms to manage, search and elaborate huge
                            collections of genomic datasets, by means of cloud and distributed technologies. He has been
                            visiting PhD candidate at Harvard University (Cambridge, MA, US). His research interests
                            include bioinformatics and computational biology, data bases and data management, big data
                            technology and algorithms, machine learning and natural language processing, and drug
                            repurposing.
                        </p>
                    </div>
                </div>
                <br><br>
            </div>
        </div>
    </div>
</section>

<section id="committee">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program Committee</h2>
                <p class="lead">
                <ul class="lead">
                    <!--<li>Raffaele Calogero, University of Turin</li>-->
                    <!--<li>Arif Canakoglu, Policlinico di Milano Ospedale Maggiore - Fondazione IRCCS Ca' Granda</li>-->
                    <!--<li>Mario Cannataro, Magna Graecia University</li>-->
                    <li>Giuseppe Agapito, Università Magna Graecia Catanzaro</li>
                    <li>José Alberto Benítez-Andrades, University of León</li>
                    <li>Stefano Ceri, Polytechnic University of Milan</li>
                    <li>Pietro Cinaglia, University Magna Graecia of Catanzaro</li>
                    <li> Tommaso Dolci, Politecnico di Milano</li>
                    <!--<li>Carlo D'Eramo, Technical University of Darmstadt</li>-->
                    <!--<li>Matteo Chiara, University of Milan</li>
                    <li>Johann Eder, University of Klagenfurt</li>-->
                    <li>Jose Luis Garrido, University of Granada</li>
                    <li>Giancarlo Guizzardi, University of Twente</li>
                    <!--<li>Carlos Íñiguez-Jarrín, National Polytechnic School</li>-->
                    <li>Ana León Palacio, Universitat Politècnica de València</li>
                    <li>Sergio Lifschitz, Pontifical Catholic University of Rio de Janeiro</li>
                    <li>Giovanni Meroni, Technical University of Denmark</li>
                    <!--<li>Paolo Missier, Newcastle University</li>-->
                    <li>Ignacio Panach, University of Valencia</li>
                    <li>Oscar Pastor, Universitat Politècnica de València</li>
                    <!--<li>Barbara Pernici, Polytechnic University of Milan</li>-->
                    <li>Rosario Michael Piro, Polytechnic University of Milan</li>
                    <!--<li>Sudha Ram, University of Arizona</li>
                    <li>Maria Rodriguez Martinez, IBM Zurich Research Laboratory</li>-->
                    <li>José Fabián Reyes Román, Universitat Politècnica de València</li>
                    <!--<li>Pasquale Stano, University of Salento</li>-->
                    <!--<li>Emanuel Weitschek, Uninettuno University</li>-->
                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<section id="invited">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Invited Talk</h2>
                <p class="text-justify">
                    <img src="imgs/oscarpastor2023.JPG" alt="oscarpastor" width="200" height="250"><br>
                    <img src="imgs/vrain.png" alt="vrain" width="200" height="90"><br>
                <h5>Oscar Pastor (Universitat Politècnica de
                    València)</h5>
                (see <a href="http://www.upv.es/ficha-personal/opastor">Research Profile</a>, <a
                    href="https://es.linkedin.com/in/opastor">LinkedIn Profile</a>)
                <br>
                <br>
                <button data-toggle="collapse" data-target="#demo"> Brief Biography</button>

                <div id="demo" class="collapse">
                    Fulll Professor and Director of the Research Center on "Métodos de Producción de Software (PROS)” at
                    the Universidad Politécnica de Valencia (Spain). He received his Ph.D. in 1992. He was a researcher
                    at HP Labs, Bristol, UK. He has published more than four hundred research papers in conference
                    proceedings, journals and books, received numerous research grants from public institutions and
                    private industry, and been keynote speaker at several conferences and workshops. Chair of the ER
                    Steering Committee, and member of the SC of conferences as CAiSE, ESEM, ICWE, CIbSE or RCIS, his
                    research activities focus on conceptual modeling, web engineering, requirements engineering,
                    information systems, and model-based software production. He created the object-oriented, formal
                    specification language OASIS and the corresponding software production method OO-METHOD. He led the
                    research and development underlying CARE Technologies that was formed in 1996. CARE Technologies has
                    created an advanced MDA-based Conceptual Model Compiler called IntegraNova, a tool that produces a
                    final software product starting from a conceptual schema that represents system requirements. He is
                    an internationally recognized researcher in the domain of Conceptual Modeling, being ER Fellow since
                    2010 and having being awarded with the Peter C. Chen in 2017. He is currently leading a
                    multidisciplinary project linking Information Systems and Bioinformatics notions, oriented to
                    designing and implementing tools for Conceptual Modeling-based interpretation of the Human Genome
                    information.
                </div>

                <br>
                <br>
                <br>
                <b>Topic:</b> Data Intensive Domains in Genomic-based Life Science: Web Engineering Implications for
                Personalized Health Care
                <br><br>
                <b>Talk abstract:</b>
                Everybody accepts that understanding the Human Genome is a big challenge for the humanity. It will take
                at the very least decades to achieve such a goal reasonably well. But new advances that are showing
                promising results come continuously. Day after day new data are is provided and new information is
                derived from them. As DNA sequencing technologies improve and evolve, it is an evidence that the rate of
                data generation at a local level is increasing dramatically. The huge amount of generated data lead to
                some sort of genomic data chaos, whose data intensive domain (DID) must be explotred. In the talk, how
                this DID-based problems must be tackled will be considered, emphasizing the role of i) Conceptual
                Modeling for data characterization, ii) Explainable AI and Machine Learning-based techniques for
                providing a sound working support to generate valuable clinical information, and iii) Web Engineering to
                provide the adequate working, practical environment. Using all these attractive software engineering,
                data science and AI related solutions, there is an opportunity to reach a fascinating scientific
                objective: understanding the secrets of life that the Genome Code hides. More and more data that relate
                genotype and phenotype are available, with especially attractive clinical applications in the context of
                Personalized Health Care. These ideas will be approached in the keynote, showing that the challenge of
                understanding the human genome can suppose a conceptual revolution: understanding the genome could allow
                to improve (even change or evolve) human being features, something that has never in our hands as
                humans. A challenging discussion is warranteed.
                </p>
            </div>
        </div>
    </div>
</section>


<section id="program">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program</h2>
                <b>WALS session is on Tuesday June 6th, 09:00-10:30 and 11.00-12.30 (CEST).
                    See the ICWE conference general program <a
                            href="https://icwe2023.webengineering.org/program/">here</a>.</b>

                <br>
                <br>
                <p class="lead">
                    09:00-09:30
                    <br>Welcome and introduction to the workshop
                    <br>
                    <br>09.30-10:30
                    <br>Oscar Pastor (<b>Keynote talk</b>):
                    <br>Data Intensive Domains in Genomic-based Life Science: Web Engineering Implications for
                    Personalized Health Care
                    <br>
                    <br>10:30-11:00
                    <br>BREAK
                    <br>
                    <br>11:00-11:30
                    <br>Babak Saremi, Mohan Xu, Chiara Tappermann and Lena Wiese:
                    <br>Development of an end-to-end web application for visualization, evaluation, and post-processing
                    of result data from neural network predictions for the melanoma use case.
                    <br>
                    <br>11:30-12:00
                    <br>Rafael García Luque, Ernesto Pimentel Sánchez and Antonio Jesús Bandera Rubio:
                    <br>Digital Avatars: An application to eHealth.
                    <br>
                    <br>12:00-12:30
                    <br>Reynaldo Alvarez, Jose Zubcoff and Irene Garrigos:
                    <br> A DCAT Profile Approach for Citizen Science.
                    <br>
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2021:</p>
                <ul class="lead">
                    <li>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez. <i>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences</i></li>

                    <li>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez. <i>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual Model-based approach</i></li>

                    <li>Federico Comolli. <i>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies</i></li>

                    <li>Sara Balderas-Díaz, Rodríguez Fórtiz María José, Gabriel Guerrero-Contreras, Mercedes Bellido-González and Jose Luis Garrido. <i>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women with SGA Foetuses</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<!--<section id="collaborations">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Collaborations</h2>
                This workshop is supported by the European Union under grant agreement No. 693174 (data-driven Genomic
                Computing)
                and by the DataMe - Spanish State Research Agency project.
                <p class="lead">
                    <a href="http://polimi.it/"><img src="imgs/polimi_white.jpg" alt="Logo CMLS" width="240"></a>
                    <a href="http://www.bioinformatics.deib.polimi.it/geco/" target="_blank"><img
                            src="imgs/geco_name.png"
                            alt="Logo GeCo"
                            width="170"
                            class="mr-4"></a>
                    <a href="http://www.upv.es/"><img src="imgs/upv_white.png" alt="Logo UPV" width="140" class="mr-4"></a>
                    <a href="http://www.pros.webs.upv.es/"><img src="imgs/pros.png" alt="Logo PROS" width="220"></a>
                </p>
            </div>
        </div>
    </div>
</section>
-->

<!-- Footer -->
<footer class="py-5 bg-dark">
    <div class="container">
        <!--<p class="m-0 text-center text-white">Copyright &copy; Your Website 2019</p>-->
        <h1 class='text-light bg-dark'>Workshop on Web Applications for Life Sciences</h1>


    </div>
    <!-- /.container -->
</footer>

<!-- Bootstrap core JavaScript -->
<script src="vendor/jquery/jquery.min.js"></script>
<script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

<!-- Plugin JavaScript -->
<script src="vendor/jquery-easing/jquery.easing.min.js"></script>

<!-- Custom JavaScript for this theme -->
<script src="../../../Downloads/startbootstrap-scrolling-nav-gh-pages/js/scrolling-nav.js"></script>

</body>

</html>
