<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Web Applications for Life Sciences</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/scrolling-nav.css" rel="stylesheet">

</head>

<body id="page-top">

<!-- Navigation -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
    <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">WALS@ICWE2022</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#topics">Topics of Interest</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#submission">Submission</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#special_issue">Special Issue</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#dates">Important Dates</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#organizers">Organizers</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#committee">Committee</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#program">Program</a>
                </li>
                <!--<li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#collaborations">Collaborations</a>
                </li>-->
            </ul>
        </div>
    </div>
</nav>

<header class="text-black-50" style="background-image: url('imgs/scenario_bg.jpg'); min-height: 27rem;">
    <div class="container text-center">
        <h1>The 1<sup>st</sup> International Workshop <br> on Web Applications for Life Sciences</h1>

        <img src="imgs/logowals2022.png" alt="Logo WALS" class="logo-cmls" width="320">

        <br><br>
        <h3>8th July, 2022 in Bari, Italy</h3>
        <!--<h4 style="color:red;opacity: 0.7;">Hybrid workshop (in person/virtually)</h4>-->
        <!--<h3 style="color:red">Held in person</h3>-->

        <br>
        <h4>In conjunction with the 22<sup>nd</sup> International Conference on Web Engineering (ICWE&nbsp;2022) - 5-8th
            July</h4>
    </div>
</header>

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">We are giving a <b>10-days extension</b> for the last ones! New hard deadline is
                    <b>August 7th 2020</b>.</p>
                <p class="text-justify lead"> Important info:
                <ul class="lead">
                    <li>All accepted papers will be published by Springer in the LNCS series in the usual book that
                        collects all ER Workshop contributions
                    </li>
                    <li>We invite all papers accepted to the workshop to submit a revised and extended version to the
                        <a href="https://bmcbioinformatics.biomedcentral.com">BMC Bioinformatics journal</a> for a post-conference
                        supplement related to Conceptual Modeling in Life Sciences.
                    </li>
                <li>For the <b>best paper of the workshop</b> (as assessed by the evaluations of the members of the Program
                        Committee) the <b>Article Processing Charges of the BMC Bioinformatics supplement will be fully
                        covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at Universitat Politècnica de València)
                    </li>
                </ul>
                </p>
                <p class="text-justify lead">
                    The 1st International Workshop on Conceptual Modeling for Life Sciences is going virtual, together
                    with the main conference <a href="https://er2020.big.tuwien.ac.at/"> ER 2020</a>.
                    We will soon provide further details on the platforms we will use and the
                    necessary logistics.
                    An important note upfront – there will be free registration for non-authors, and a heavily reduced
                    registration fee for authors presenting a paper. Registration with further details will be open on
                    August 15, 2020.</p>
            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead">Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>
                <p class="text-justify lead"> We made a compilation of the workshop presentations. You can watch it <a href="https://www.youtube.com/watch?v=-l4OUdrkVC4&list=PLfWxoOMC6swI59BOYNw0U3L2VNJ_dqSBG&ab_channel=bioinformaticspolimi">here on YouTube</a>.
                <p class="text-justify lead">Check also the Program (at the end of the page) for slides and videos of each video (we will keep loading them as we get permission from the authors).</p>

            </div>

        </div>
    </div>
</section>-->

<!--<section id="latest">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Latest news</h2>
                <p class="text-justify lead"><b>November 4th, 2021</b>: our BMC Bioinformatics Supplement on CMLS 2020 (Part 1) is finally out!
    Check it out here <a
                        href="https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13">
                    https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-22-supplement-13</a>.
                Keep connected to know when also Part 2 is out. Also for CMLS 2021 we are planning a supplement on
                    the same Journal. The deadline for extended articles (invited) is Jan. 31st, 2022.</p>
                 <p class="text-justify lead"><b>October 19th, 2021</b>: Thank you for everyone who joined the workshop and to all presenters for sharing their interesting research!</p>

            </div>

        </div>
    </div>
</section>-->


<section id="about">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>About</h2>
                <p class="text-justify lead">
                    The recent advances in unraveling the secrets of human conditions and diseases have encouraged new
                    paradigms for their diagnosis and treatment. In addition, the COVID-19 pandemic has attracted
                    increasing attention toward the genetic mechanisms of viruses and infected hosts. The information
                    related to these phenomena is increasing at unprecedented rates, directly impacting the design and
                    development of data management pipelines and their applications. New ways of processing and exposing
                    data and knowledge on the Web in healthcare and life sciences environments are thus strongly needed.
                    <br><br>
                    The WALS workshop aims at being an initial meeting forum for Web Engineering, Data Management, and
                    Bioinformatics researchers working on life sciences problems, offering the opportunity to share,
                    discuss and find new approaches to support Search, Knowledge Extraction, and Data Science on the
                    Web, thereby achieving important results for healthcare, precision medicine, biology, genomics, and
                    virology.
                    <br><br>
                    The workshop welcomes submissions of fresh investigations and demonstrations concerning conceptual,
                    experimental, and applied studies of life sciences problems on the Web. Directions of particular
                    interest include: 1) monitoring and adaptation of Web Engineering techniques for life sciences; 2)
                    design and engineering of Web applications for resolving life sciences problems; 3) development of
                    crowdsourcing and collaborative strategies for life sciences data curation, integration, and
                    discovery.

                </p>
            </div>
        </div>
    </div>
</section>

<section id="topics">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Topics of interest</h2>
                <p class="text-justify lead">
                    The workshop focuses on Web Engineering as a means for facing the challenges that emerge when
                    designing and developing applications for life sciences, focused on several branches such as
                    healthcare, precision medicine, molecular biology, genomics, and virology. The workshop is not
                    restricted to particular research methods, and we will consider conceptual, empirical, and applied
                    research (including practical demonstrations). The topics of interest include, but are not limited
                    to:


                </p>
                <p class="text-justify lead">The topics of interest include, but are not limited to:</p>
                <ul class="lead">
                    <li>Web applications modeling and engineering for genomics</li>
                    <li>Web applications for the management of electronic/digital health records (EHR)</li>
                    <li>Web user interfaces for genomics, virology, or molecular biology</li>
                    <li>Development of Web systems for Precision Medicine</li>
                    <li>Web services for exploration of life sciences databases and knowledge graphs</li>
                    <li>Biological knowledge extraction</li>
                    <li>Genomic big data analytics</li>
                    <li>Web crowdsourcing methods for life sciences data curation and annotation</li>
                    <li>Information visualization and exploratory analysis methods for biology</li>
                    <li>Data science applications to healthcare and biology on the Web</li>
                    <li>Semantic Web knowledge representation for genomics</li>
                    <li>Life sciences Semantic Web applications</li>
                    <li>Biomedical ontologies engineering</li>
                    <li>Interoperability of health information systems and databases on the Web</li>
                    <li>Web Engineering techniques for FAIRification of biological datasets</li>
                    <li>COVID-19 and SARS-CoV-2 tracking via Web applications</li>
                </ul>
            </div>
        </div>
    </div>
</section>

<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for WALS 2022:</p>
                <ul class="lead">
                    <li>Marco Antonio Tangaro, Marica Antonacci, Pietro Mandreoli, Daniele Colombo, Nadina Foggetti,
                        Giacinto Donvito, Graziano Pesole and Federico Zambelli.
                        <i>The Laniakea Dashboard and storage encryption components: a foundation for developing
                            on-demand cloud services for Life Science information on submission</i></li>
                    <li>Alberto García, Ana León, Mireia Costa, Oscar Pastor, Daniel González-Ibea, Estela Pérez-Román,
                        Carles Borredà, Javier Terol, Victoria Ibanez, Francisco R. Tadeo and Manuel Talón.
                        <i>CitrusGenome: A Bioinformatics Tool to Characterize, Visualize, and Explore Large Citrus
                            Variant
                            Datasets information on submission</i></li>
                    <li>Paolo Perlasca, Marco Frasca, Cheick Tidiane Ba, Jessica Gliozzo, Marco Notaro, Mario
                        Pennacchioni,
                        Giorgio Valentini and Marco Mesiti.
                        <i>Integration and Visual Analysis of Biomolecular Networks</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>

<section id="submission">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Paper submission guidelines</h2>
                <p class="text-justify lead">
                    We invite submissions of high quality papers describing original and unpublished results
                    regarding any of the workshop’s topics of interest.
                    <br><br>
                    Accepted papers will be included in the post-workshop proceedings to appear in Springer's series.

                    The authors must submit manuscripts using the Springer-Verlag LNCS style for Lecture Notes in
                    Computer Science.
                    For style files and details, see the page <a
                        href="http://www.springer.de/comp/lncs/authors.html">http://www.springer.de/comp/lncs/authors.html</a>.
                    The workshop will accept two types of contributions:
                <ul class="lead">
                    <li>Full papers - maximum 12 pages including references</li>
                    <li>Short/demo papers - maximum 6 pages including references</li>
                </ul>
                <p class="text-justify lead">
                    Papers must be submitted as PDF files using EasyChair at <a
                        href="https://easychair.org/conferences/?conf=wals2022" target="_blank">https://easychair.org/conferences/?conf=wals2022</a>.
                    <br><br>
                    To ensure high quality, all papers will be thoroughly peer reviewed by the Program Committee.
                    Manuscripts not submitted in the LNCS style or having more than 12 pages will be automatically
                    rejected.
                    The papers need to be original and not submitted or accepted for publication in any other
                    workshop, conference, or journal.
                    <br><br>
                    For each accepted paper, at least one author must attend the workshop and present the paper.
                    Registration is subject to the terms and conditions of ICWE.
                    <br><br>
                    Contact: anna.bernasconi@polimi.it
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="cameraready">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <strike><h2>Camera-ready guidelines</h2></strike>
                <strike><p class="text-justify lead">
                    For the preparation of their manuscript camera-ready version, authors should consult <a
                        href="ftp://ftp.springernature.com/cs-proceeding/svproc/guidelines/Springer_Guidelines_for_Authors_of_Proceedings.pdf">Springer’s
                    authors’ guidelines</a>
                    and use their <a
                        href="https://www.springer.com/de/it-informatik/lncs/conference-proceedings-guidelines">proceedings
                    templates</a>, either for LaTeX or for Word,
                    for the preparation of their papers. The page limit for workshop papers is 10 pages, as for the
                    previously submitted version.
                    Springer encourages authors to include their ORCIDs in their papers.
                    <br><br>
                    In addition, the corresponding author of each paper, acting on behalf of all of the authors of that
                    paper,
                    must complete and sign a <a
                        href="https://drive.google.com/file/d/1FL5_jFrqayqbPRZ2nTuNKaY55_8L8xD8/view?usp=sharing">Consent-to-Publish
                    form</a>.
                    The corresponding author signing the copyright form
                    should match the corresponding author marked on the paper.
                    Once the files have been sent to Springer, changes relating to the authorship of the papers cannot
                    be made.
                </p></strike>
            </div>
        </div>
    </div>
</section>
-->


<section id="special_issue">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Post-conference publication</h2>
                <p class="text-justify lead">
                    <!--The papers accepted to the workshop's last edition CMLS 2020,
                    published within the Springer volume <a
                        href="https://link.springer.com/book/10.1007/978-3-030-65847-2">Advances in Conceptual
                    Modeling</a>,
                    have been invited to submit a revised and extended version
                    for a post-conference supplement in the journal
                    <a
                            href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                        Bioinformatics</b></a>
                    journal (2-year IF 2.511, SJR 1.374).
                    We are working towards an agreement with the same journal also for this year.-->
                    We are working toward a preliminary agreement with <a
                        href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                    Bioinformatics</b></a>
                    journal (2-year IF 2.511, SJR 1.374)
                    for a <b>post-conference supplement</b> related to Web Applications in Life Sciences.
                    High quality papers accepted to our workshop will be invited
                    to submit a revised and extended version to the journal supplement.
                </p>
                <!-- <p class="text-justify lead">For the <b>best paper</b> of the workshop (as assessed by the evaluations
                     of the members of the Program Committee) the <b>Article Processing Charges of the BMC Bioinformatics
                         supplement will be fully covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at
                     Universitat Politècnica de València).
                 </p>-->
            </div>
        </div>
    </div>
</section>


<section id="dates">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Important dates (AoE)</h2>
                <p class="lead">
                <ul class="lead">
                    <!--<li><strike> Paper submission: July 9th, 2021 / July 16th, 2021 (extended)</strike></li>
                    <li><strike>Notification: August 6th, 2021</strike></li>
                    <li><strike>Camera-ready version: August 20th, 2021</strike></li>
                    <li><strike>ER Conference dates: October 18-21st, 2021. CMLS Workshop date: October 19th</strike>
                    </li>
                    <li>Extended article submission to BMC Bioinformatics supplements: January 31st, 2022</li>-->

                    <li>Paper submission: <strike>April 24th, 2022, <b>May 5th, 2022 (EXTENDED!)</b></strike></li>
                    <li>Notification: <strike>May 31st, 2022</strike></li>
                    <li>Workshop day: July 8th</li>
                    <li>Camera-ready version: July 29th, 2022</li>
                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<section id="registration">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Registration</h2>
                <p class="lead">
                    The registration for the workshop is open via the <a
                        href="https://icwe2022.webengineering.org/registration/">ICWE website</a>.
                    At least one of the authors for each accepted paper must have a "Regular Registration"
                    and attend the conference for paper presentation.
                </p>
            </div>
        </div>
    </div>
</section>

<section id="organizers">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Organizers</h2>
                <p class="lead">
                    <b>Anna Bernasconi</b>
                    <br>Department of Electronics, Information, and Bioengineering - Politecnico di Milano, Italy
                    <br>anna.bernasconi@polimi.it
                </p>
                <p class="text-justify">
                    Anna Bernasconi obtained her PhD cum laude at Politecnico di Milano, within the “Data-driven Genomic
                    Computing” ERC Awarded project (2016-2021), under the supervision of Professor Stefano Ceri. She is
                    now a postdoctoral researcher in the Department of Electronics, Information, and Bioengineering also
                    at Politecnico di Milano. Her research focuses on conceptual modeling, data integration, semantic
                    web, and biological data analysis. Since the COVID-19 pandemic, her research has moved to viral
                    genomics, by building models, databases, and Web search systems for viral sequences and their
                    variants.
                <p><br><br>
                <p class="lead">
                    <b>Giuseppe Agapito</b>
                    <br>Department of Law, Economics and Social Sciences - "Magna Græcia" University of Catanzaro,
                    Italy.
                    <br>agapito@unicz.it
                </p>
                <p class="text-justify">
                    Giuseppe Agapito is an assistant professor and a senior research scientist in the field of Parallel
                    and Distributed Computing, Machine Learning, Omics and Biological Networks data Analysis at the
                    University Magna Græcia, of Catanzaro, Italy.
                    His research interests are focused on the study of machine learning methods that can be used to take
                    advantage of the vast amount of data that are produced nowadays. In particular, the research focuses
                    on the development, implementation, and application of computational intelligence techniques for
                    addressing complex real-world problems in different domains, especially in the field of biology and
                    omics sciences.
                </p><br><br>

                <p class="lead">
                    <b>Luca Nanni</b>
                    <br>Department of Computational Biology (DBC) - University of Lausanne, Switzerland
                    <br>luca.nanni@unil.ch
                </p>
                <p class="text-justify">
                    Luca Nanni works as a Postdoctoral fellow in the Computational Systems Oncology laboratory at
                    University of Lausanne, led by Professor Giovanni Ciriello. He received his PhD cum laude in 2021 in
                    Computer Science and Engineering at Politecnico di Milano, Italy, under the supervision of Professor
                    Stefano Ceri. His work focuses on the modeling of DNA conformation in the three-dimensional space
                    using statistical and algorithmic methodologies, with a specific interest in cancer and aneuploid
                    genomes. His work lies at the intersection of computer science, bioinformatics, computational
                    biology and statistics and big data management.
                </p><br><br>

                <p class="lead">
                    <b>Pietro Pinoli</b>
                    <br>Department of Electronics, Information, and Bioengineering - Politecnico di Milano, Italy
                    <br>pietro.pinoli@polimi.it
                </p>
                <p class="text-justify">
                    Pietro Pinoli works as Researcher Fellow and lecturer at the Department of Electronics, Information
                    and Bioengineering at the Politecnico di Milano (Italy). He received his PhD cum laude in 2017, with
                    a thesis titled “Modeling and Querying Genomic Data” where he proposed and benchmarked data
                    structures and algorithms to manage, search and elaborate huge collections of genomic datasets, by
                    means of cloud and distributed technologies. He has been visiting PhD candidate at Harvard
                    University (Cambridge, MA, US). His research interests include bioinformatics and computational
                    biology, data bases and data management, big data technology and algorithms, machine learning and
                    natural language processing, and drug repurposing.

                </p>
            </div>
        </div>
    </div>
</section>

<section id="committee">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program Committee</h2>
                <p class="lead">
                <ul class="lead">
                    <!--<li>Raffaele Calogero, University of Turin</li>-->
                    <li>Arif Canakoglu, Policlinico di Milano Ospedale Maggiore - Fondazione IRCCS Ca' Granda</li>
                    <li>Mario Cannataro, Magna Graecia University</li>
                    <li>Stefano Ceri, Polytechnic University of Milan</li>
                    <li>Carlo D'Eramo, Technical University of Darmstadt</li>
                    <!--<li>Matteo Chiara, University of Milan</li>
                    <li>Johann Eder, University of Klagenfurt</li>-->
                    <li>Jose Luis Garrido, University of Granada</li>
                    <li>Giancarlo Guizzardi, University of Twente</li>
                    <!--<li>Carlos Íñiguez-Jarrín, National Polytechnic School</li>-->
                    <li>Ana León Palacio, Universitat Politècnica de València</li>
                    <li>Sergio Lifschitz, Pontifical Catholic University of Rio de Janeiro</li>
                    <!--<li>Paolo Missier, Newcastle University</li>-->
                    <!--<li>Ignacio Panach, University of Valencia</li>-->
                    <li>Oscar Pastor, Universitat Politècnica de València</li>
                    <!--<li>Barbara Pernici, Polytechnic University of Milan</li>-->
                    <li>Rosario Michael Piro, Polytechnic University of Milan</li>
                    <!--<li>Sudha Ram, University of Arizona</li>
                    <li>Maria Rodriguez Martinez, IBM Zurich Research Laboratory</li>-->
                    <li>José Fabián Reyes Román, Universitat Politècnica de València</li>
                    <li>Pasquale Stano, University of Salento</li>
                    <li>Emanuel Weitschek, Uninettuno University</li>
                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<section id="invited">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Invited Talk</h2>
                <p class="text-justify">
                    <a
                            href="https://scholar.google.it/citations?user=PLfNQvAAAAAJ&hl=it">
                        Raffaele A. Calogero</a> (Dept. of Molecular Biotechnology and Health Sciences, University of
                    Torino).
                    <button data-toggle="collapse" data-target="#demo"> Brief Biography</button>

                <div id="demo" class="collapse">
                    My research experience was devoted to the development and optimization of analysis workflows and in
                    mining transcription-based experiments, mainly in the oncology framework. An important aspect of my
                    past and present work was the constant focus on new technologies (e.g. today single-cell and
                    single-molecule sequencing) and their integration in our analysis workflows.
                    <br>
                    I started my journey in Bioinformatics in latest 90’s when I developed a tool to simulate, for
                    windows-based PCs, a molecular biology laboratory (Iazzetti, et al., 1998). In 1998, I established
                    at University of Torino the Genomics and Bioinformatics unit (B&Gu). B&Gu is an interdisciplinary
                    group devoted to the study of multifactorial diseases by mean of high throughput technologies - i.e.
                    microarray, Next Generation Sequencing – and bioinformatics. Since then my interests moved to
                    research topics in which computational approaches could be used to mine biological data. Those years
                    were the period in which expression microarrays started to be an effective tool to investigate genes
                    involved in diseases and general biological processes. I was involved in various studies in which
                    microarray data analysis played an important role. In collaboration with the group of cancer
                    immunologists leaded by Prof. Forni at University of Torino, I focused the work of my group on the
                    identification ofnew targets for anti-tumor vaccination protocols Because of my experience in
                    microarray data analysis, my group was involved in a large number of collaborations in Italy and
                    abroad. It is notable that, using gene expression microarrays, we were able to identify for the
                    first time the presence of coding transcripts in circulating blood vesicles (Bruno,et al., 2009;
                    Deregibus, et al., 2007; Herrera, et al., 2010), today a very hot topic in the field of biomarker
                    discovery.
                    <br>
                    More recently we move our interest to single-cell transcriptomics both focusing in tool development
                    (Alessandrì et al. Gigascience 2019) and data analysis (Rodriguez-Fraticelli et al. Nature 2018).
                    <br>
                    Since I started my research career as molecular biologist, I am convinced that
                    genomics/transcriptomics data analysis, at least to some extent, should be placed back in the hands
                    of wet biologists, thus itis more than 20 years that I am involved in training of PhD and Post-docs
                    int he transcriptomics field, running courses at University of Naples first, thenat University of
                    Turin (Sponsored by Affimetrix first and later by lllumina), at EMBL-Heidelberg, DUKE-NUS in
                    Singapore and Italian Institute of Technology in Genova. Thus, with my group I have developed tools
                    to simplify data analysis for microarray first (Sanges, et al., 2007) and then for Next generation
                    sequencing (Beccuti, et al., 2017; Beccuti, et al., 2018) and single-cell transcriptomics today
                    (Alessandrì et al. 2019). I am also the cofounder of the Reproducible Bioinformatics Project
                    (Kulkarni et al. 2018), a community of bioinformaticians aiming to the creation of easy to use and
                    reproducible bioinformatics workflows (http://www.reproducible-bioinformatics.org/).
                </div>

                <br>
                <b>Topic:</b> Are web applications fulfilling the needs of life science scientists?
                <br><br>
                <b>Talk abstract:</b>
                Since biology and medicine are disciplines generating a massive amount of complex and
                hard-to-decipher data, bioinformatics is pervading the life science fields. Consequently,
                bioinformatics is a constantly growing field, producing a massive amount of computing tools.
                However, life science suffers from the lack of an adequate number of bioinformaticians. Furthermore,
                the number of life scientists understanding the need of building a background in basic programming
                is still very limited. The critical points highlighted above, taken together with the limited access
                to computing resources for many life scientists, have favored the development of a lot of web-based
                bioinformatics applications.
                <br>
                In this presentation, I wish to focus on the state of the art of web applications for life science
                and discuss their strength and limits. Furthermore, I wish to propose the possibility to move to
                more advanced web applications, where users can interact with them in a more sophisticated way, but
                without the need of acquiring coding skills, e.g. using syntax-free programming or a no-code
                approach.
                </p>
            </div>
        </div>
    </div>
</section>


<section id="program">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program</h2>

                <b>WALS session is on Friday July 8th, 09:00-10:30 (CEST) and 11.00-12.30.
                    See the ICWE conference general program <a
                            href="https://icwe2022.webengineering.org/programandsessions/">here</a>.</b>

                <br>
                <br>
                <p class="lead">
                    09:00-09:30
                    <br>Welcome and introduction to the workshop
                    <br>
                    <br>09.30-10:30
                    <br>Raffaele A. Calogero (<b>Keynote talk</b>):
                    <br>Are web applications fulfilling the needs of life science scientists?
                    <br>
                    <br>10:30-11:00
                    <br>BREAK
                    <br>
                    <br>11:00-11:30
                    <br>Alberto García, Ana León, Mireia Costa, Oscar Pastor, Daniel González-Ibea, Estela Pérez-Román,
                    Carles Borredà, Javier Terol, Victoria Ibanez, Francisco R. Tadeo and Manuel Talón:
                    <br>CitrusGenome: A Bioinformatics Tool to Characterize, Visualize, and Explore Large Citrus Variant
                    Datasets information on submission.
                    <br>
                    <br>11:30-12:00
                    <br>Marco Antonio Tangaro, Marica Antonacci, Pietro Mandreoli, Daniele Colombo, Nadina Foggetti,
                    Giacinto Donvito, Graziano Pesole and Federico Zambelli:
                    <br>The Laniakea Dashboard and storage encryption components: a foundation for developing on-demand
                    cloud services for Life Science information on submission.
                    <br>
                    <br>12:00-12:30
                    <br>Paolo Perlasca, Marco Frasca, Cheick Tidiane Ba, Jessica Gliozzo, Marco Notaro, Mario
                    Pennacchioni, Giorgio Valentini and Marco Mesiti:
                    <br> Integration and Visual Analysis of Biomolecular Networks.
                    <br>
                </p>
            </div>
            -->
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2021:</p>
                <ul class="lead">
                    <li>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez. <i>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences</i></li>

                    <li>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez. <i>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual Model-based approach</i></li>

                    <li>Federico Comolli. <i>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies</i></li>

                    <li>Sara Balderas-Díaz, Rodríguez Fórtiz María José, Gabriel Guerrero-Contreras, Mercedes Bellido-González and Jose Luis Garrido. <i>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women with SGA Foetuses</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<!--<section id="collaborations">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Collaborations</h2>
                This workshop is supported by the European Union under grant agreement No. 693174 (data-driven Genomic
                Computing)
                and by the DataMe - Spanish State Research Agency project.
                <p class="lead">
                    <a href="http://polimi.it/"><img src="imgs/polimi_white.jpg" alt="Logo CMLS" width="240"></a>
                    <a href="http://www.bioinformatics.deib.polimi.it/geco/" target="_blank"><img
                            src="imgs/geco_name.png"
                            alt="Logo GeCo"
                            width="170"
                            class="mr-4"></a>
                    <a href="http://www.upv.es/"><img src="imgs/upv_white.png" alt="Logo UPV" width="140" class="mr-4"></a>
                    <a href="http://www.pros.webs.upv.es/"><img src="imgs/pros.png" alt="Logo PROS" width="220"></a>
                </p>
            </div>
        </div>
    </div>
</section>
-->

<!-- Footer -->
<footer class="py-5 bg-dark">
    <div class="container">
        <!--<p class="m-0 text-center text-white">Copyright &copy; Your Website 2019</p>-->
        <h1 class='text-light bg-dark'>Workshop on Web Applications for Life Sciences</h1>


    </div>
    <!-- /.container -->
</footer>

<!-- Bootstrap core JavaScript -->
<script src="vendor/jquery/jquery.min.js"></script>
<script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

<!-- Plugin JavaScript -->
<script src="vendor/jquery-easing/jquery.easing.min.js"></script>

<!-- Custom JavaScript for this theme -->
<script src="../../../Downloads/startbootstrap-scrolling-nav-gh-pages/js/scrolling-nav.js"></script>

</body>

</html>
